### PR TITLE
[Module 2] Lab3 done

### PR DESCRIPTION
**Pull request by fhtrancoso**
- Identifying, for each hour, the driver earning the most tips.

### CODE REVIEW

File: ride-cleansing/src/main/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingExercise.java
Comment (ness-PavelKhokhlov) [inline ride-cleansing/src/main/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingExercise.java:84]: I thought I approved that in previous PR, not sure why I should review it second time.
You just need to apply first PR
https://github.com/fhtrancoso/flink-training/pull/1
and then build next lab on top of it.

**Review by ness-PavelKhokhlov — APPROVED** [review]

File: ride-cleansing/src/main/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingExercise.java
Comment (fhtrancoso) [reply ride-cleansing/src/main/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingExercise.java:84]: Because I did not merge the first PR. I did it now. And will do in this one.

**Pull request merged**
